amr-wind/wind_energy/actuator/wing/wing_ops.H

```diff
@@ -186,6 +232,9 @@ struct ComputeForceOp<
             // Compute force on fluid from this section of wing
             grid.force[ip] = -(lift_dir * lift + drag * drag_dir);
 
+            // In global coords, zero disabled force components
+            grid.force[ip] = grid.force[ip] * fcoord_flags;
```

Comment:
could be a `*=`

Reply:
avoided this initially because the vs::Vector type didn't have it as an operator. added it now